files/zh-cn/learn/forms/how_to_build_custom_form_controls/example_3/index.md

```diff
@@ -167,80 +167,86 @@ original_slug: Learn/HTML/Forms/How_to_build_custom_form_widgets/Example_3
 // ------- //
 
 NodeList.prototype.forEach = function (callback) {
-  Array.prototype.forEach.call(this, callback);
+    Array.prototype.forEach.call(this, callback);
```

Comment:
It seems that this content is not synced with english document?

And we just use two spaces as indentation

<https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_build_custom_form_controls/Example_3>